### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by waynew**
Added all relevant changes from the PRs that were merge to master.

Only 457 to review :joy: :weary:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,43 +3,246 @@ All notable changes to Salt will be documented in this file.
...
+- [#52764](https://github.com/saltstack/salt/pull/52764) via [#54058](https://github.com/saltstack/salt/pull/54058) - Added vSphere tagging ability - [@xeacott](https://github.com/xeacott)
+- [#53307](https://github.com/saltstack/salt/pull/53307) - Added slot parsing inside nested state data structures - [@max-arnold](https://github.com/max-arnold)
+- [#53621](https://github.com/saltstack/salt/pull/53621) - Added support for `git_pillar_update_interval` - [@sathieu](https://github.com/sathieu)
 - [#53736](https://github.com/saltstack/salt/issues/53736) - Added index get_settings, put_settings methods for Elasticsearch module. - [@Oloremo](https://github.com/Oloremo)
+- [#53738](https://github.com/saltstack/salt/pull/53738) - Added `request_interval` feature to `http.wait_for_successful_query` module - [@Oloremo](https://github.com/Oloremo)
+- [#53738](https://github.com/saltstack/salt/pull/53738) - Added `request_interval` feature to `http.wait_for_successful_query` module - [@Oloremo](https://github.com/Oloremo)
```
Comment (Oloremo) [inline CHANGELOG.md]: duplicate

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,43 +3,246 @@ All notable changes to Salt will be documented in this file.
...
+- [#55506](https://github.com/saltstack/salt/pull/55506) - Added `hashutil.hmac_compute` - [@Ajnbro](https://github.com/Ajnbro)
+- [#55515](https://github.com/saltstack/salt/pull/55515) - Added `disk_set` and `disk_toggle` to parted module - [@aplanas](https://github.com/aplanas)
+- [#55516](https://github.com/saltstack/salt/pull/55516) - Added `not_change` to several functions in the mount module, `set_fstab` & others - [@aplanas](https://github.com/aplanas)
+- [#55565](https://github.com/saltstack/salt/pull/55565) - Added ability to pass the context dictionary to Sminion and Runner - [@s0undt3ch](https://github.com/s0undt3ch)
+- [#55590](https://github.com/saltstack/salt/pull/55590) - Added `version` to depends decorator - [@github-abcde](https://github.com/github-abcde)
+- [#55593](https://github.com/saltstack/salt/issues/55593) - Added a support for a global proxy to pip module. - [@Oloremo](https://github.com/Oloremo)
+- [#55596](https://github.com/saltstack/salt/pull/55596) - Added support for salt global proxy to pip module - [@Oloremo](https://github.com/Oloremo)
```
Comment (Oloremo) [inline CHANGELOG.md]: duplicate

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,43 +3,246 @@ All notable changes to Salt will be documented in this file.
...
+- [#55613](https://github.com/saltstack/salt/pull/55613) - Added saltcheck updates for Neon - [@mchugh19](https://github.com/mchugh19)
+- [#55636](https://github.com/saltstack/salt/pull/55636) - Added DSON outputter - [@terminalmage](https://github.com/terminalmage)
+- [#55637](https://github.com/saltstack/salt/pull/55637) - Added wildcard matches and grains matching to `config.option` - [@terminalmage](https://github.com/terminalmage)
+- [#55639](https://github.com/saltstack/salt/pull/55639) - Added `loop.until_no_eval` - [@github-abcde](https://github.com/github-abcde)
+- [#55666](https://github.com/saltstack/salt/pull/55666) - Added the `internal` flag to openvswitch - [@Akm0d](https://github.com/Akm0d)
+- [#55700](https://github.com/saltstack/salt/pull/55700) - Added a `skip_files_list_verify` argument to archive.extracted state - [@Oloremo](https://github.com/Oloremo)
```
Comment (Oloremo) [inline CHANGELOG.md]: duplicate

Comment (waynew) [reply CHANGELOG.md]: Got 'em, thanks for the review! 🙌

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,43 +3,243 @@ All notable changes to Salt will be documented in this file.
...
+- [#54991](https://github.com/saltstack/salt/pull/54991) - Added keystore state and modules for Java keystore files - [@mchugh19](https://github.com/mchugh19)
+- [#54992](https://github.com/saltstack/salt/pull/54992) - Added ability to use salt modules in onlyif and unless - [@mchugh19](https://github.com/mchugh19) and [@gtmanfred](https://github.com/gtmanfred)
+- [#54993](https://github.com/saltstack/salt/pull/54993) - Added support for parsing slot results - [@mchugh19](https://github.com/mchugh19)
+- [#54996](https://github.com/saltstack/salt/pull/54996) - Added `binds` parameter for `run_chroot` - [@aplanas](https://github.com/aplanas)
+- [#55001](https://github.com/saltstack/salt/pull/55001) - Added ability to ignore errors on `mdadm_raid.examine` - [@aplanas](https://github.com/aplanas)
+- [#55047](https://github.com/saltstack/salt/pull/55047) - Added ability to deprecate by date - [@s0undt3ch](https://github.com/s0undt3ch)
+- [#55145](https://github.com/saltstack/salt/pull/55145) - Added status code lists for `http.query` state - [@Ajnbro](https://github.com/Ajnbro)
```
Comment (mchugh19) [inline CHANGELOG.md]: This change also includes 50150. Should read 
> Added status code lists and status regex for `http.query` state - [@Ajnbro](https://github.com/Ajnbro) and [@mchugh19](https://github.com/mchugh19)

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,43 +3,243 @@ All notable changes to Salt will be documented in this file.
...
+- [#55195](https://github.com/saltstack/salt/pull/55195) - Added `salt_version` module - [@rallytime](https://github.com/rallytime) and [@max-arnold](https://github.com/max-arnold)
+- [#55200](https://github.com/saltstack/salt/pull/55200) - Added `virt.pool_deleted` state - [@cbosdo](https://github.com/cbosdo)
+- [#55202](https://github.com/saltstack/salt/pull/55202) - Added test ability and pool editing to `virt.pool_running` - [@cbosdo](https://github.com/cbosdo)
+- [#55203](https://github.com/saltstack/salt/pull/55203) - Adds enabled kwarg to `aptpkg` module - [@brejoc](https://github.com/brejoc)
+- [#55245](https://github.com/saltstack/salt/pull/55245) - Adding kernel boot parameters to libvirt xml - [@ldeweysuse](https://github.com/ldeweysuse)
+- [#55256](https://github.com/saltstack/salt/pull/55256) - Added status to dpkg.info response - [@mateiw](https://github.com/mateiw)
+- [#55342](https://github.com/saltstack/salt/pull/55342) - Added Slack webhook returner - [@cdalvaro](https://github.com/cdalvaro)
+- [#55345](https://github.com/saltstack/salt/pull/55345) - Add chroot apply_, sls, and highstate for state execution - [@aplanas](https://github.com/aplanas)
```
Comment (mchugh19) [inline CHANGELOG.md]: This should also get an entry in the release notes

Comment (waynew) [reply CHANGELOG.md]: I'm not super familiar with that feature - added it to the release notes based on what I understood from the docs. Let me know if you think it needs any massaging. Thanks for bringing that up!

**Review by Akm0d — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

File: doc/topics/releases/3000.rst [hunk 6]
```diff
@@ -528,11 +483,13 @@ the ``beacon_module`` parameter in the beacon configuration.
             /etc/another_file: {}
         - beacon_module: inotify
 
-Module Changes
-==============
+Enhancements to chroot
+======================
 
-- Added new :py:func:`boto_ssm <salt.modules.boto_ssm>` module to set and query
```
Comment (mchugh19) [inline doc/topics/releases/3000.rst]: I think this is in Neon with https://github.com/saltstack/salt/pull/54982

Comment (waynew) [reply doc/topics/releases/3000.rst]: Yes - it was just in the wrong place in the releasenotes and duplicated :trollface: 

It's still here in the right place: https://github.com/saltstack/salt/pull/55882/files/28808792d85f88e6a5cf66629c1274ebf95dd037#diff-0ef54636435038ddec39952d7e8fe37bR418

Comment (waynew) [reply doc/topics/releases/3000.rst]: Yes - it was just in the wrong place in the releasenotes and duplicated :trollface: 

It's still here in the right place: https://github.com/saltstack/salt/pull/55882/files/28808792d85f88e6a5cf66629c1274ebf95dd037#diff-0ef54636435038ddec39952d7e8fe37bR418

**Review by rares-pop — APPROVED** [review]

**Pull request merged**